deploy/005-deploy-VTreasuryV8.ts

```diff
@@ -0,0 +1,58 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+interface AdminAccounts {
+  [key: string]: string;
+}
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const getTimelock = async () => {
+    switch (hre.network.name) {
+      case "bsctestnet":
+      case "bscmainnet": {
+        const timelock = await deployments.get("NormalTimelock");
+        return timelock.address;
+      }
+    }
+    return "";
+  };
+
+  const acmAdminAccount: AdminAccounts = {
+    sepolia: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb", // SEPOLIA MULTISIG
+    ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67", // ETHEREUM MULTISIG
+    opbnbtestnet: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf", // OPBNBTESTNET MULTISIG
+    opbnbmainnet: "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207", // OPBNBMAINNET MULTISIG
+    bscmainnet: await getTimelock(),
+    bsctestnet: await getTimelock(),
+    hardhat: deployer,
+  };
+
+  const deployerSigner = await hre.ethers.getSigner(deployer);
+  const treasuryInstance = await deploy("VTreasury", {
```

Comment:
This is changing the name of the deployed contract, from `VTreasuryV8` to `VTreasury`. It's a breaking change. Do we need it? I think I would keep the `VTreasuryV8` name, but I would like to listen other opinions

Reply:
The problem is that this contract was originally deployed as VTreasury and then later was deployed as VTreasuryV8. I don't have a preference. I went with the earlier name although it created more changes since all networks after use V8